### ISSUE TITLE
Experiencing `Cannot read property 'originalPositionFor' of undefined` in `jest`

### ISSUE THREAD

**Issue by oliversalzburg** [ampproject/remapping#162]
I was wondering if this is an issue that is relevant at the root, even though I'm experiencing it with Jest.

When I try to collect coverage with Jest, it will fail hard with some of my files and `remapping` is what is complaining about my files:
```shell
ERROR: /home/oliver/fairmanager/core-web3/packages/core-graphql/source/edges/IndexedUserProfileFields.ts: Cannot read property 'originalPositionFor' of undefined
STACK: TypeError: /home/oliver/fairmanager/core-web3/packages/core-graphql/source/edges/IndexedUserProfileFields.ts: Cannot read property 'originalPositionFor' of undefined
    at SourceMapTree.originalPositionFor (/home/oliver/fairmanager/core-web3/node_modules/@ampproject/remapping/dist/remapping.umd.js:159:27)
    at trace (/home/oliver/fairmanager/core-web3/node_modules/@ampproject/remapping/dist/remapping.umd.js:102:37)
    at EncodedSourceMapImpl.map (/home/oliver/fairmanager/core-web3/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:347:36)
    at TraceMap.map (/home/oliver/fairmanager/core-web3/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:430:31)
    at SourceMapTree.traceMappings (/home/oliver/fairmanager/core-web3/node_modules/@ampproject/remapping/dist/remapping.umd.js:85:34)
    at remapping (/home/oliver/fairmanager/core-web3/node_modules/@ampproject/remapping/dist/remapping.umd.js:258:36)
    at mergeSingleSource (/home/oliver/fairmanager/core-web3/node_modules/@babel/core/lib/transformation/file/merge-map.js:42:22)
    at mergeSourceMap (/home/oliver/fairmanager/core-web3/node_modules/@babel/core/lib/transformation/file/merge-map.js:31:14)
    at generateCode (/home/oliver/fairmanager/core-web3/node_modules/@babel/core/lib/transformation/file/generate.js:75:39)
    at run (/home/oliver/fairmanager/core-web3/node_modules/@babel/core/lib/transformation/index.js:55:33)
```

The file (`IndexedUserProfileFields.ts`) it complains about only exports types, which I assume is the issue:
```ts
export type IndexedUserProfileFields = {
  costUnit: string;
  country: string;
  email: string;
  nameFirst: string;
  nameLast: string;
};
```

I investigated this a bit at the source level and was wondering what `segment` is if `source` is undefiend. In this error situation, it's `[,NaN,NaN,NaN,NaN]`. It didn't seem right to me and maybe this is a case that already needs to be handled here.

**Comment by jridgewell** [comment]
Hi @oliversalzburg: Is there a chance you still have `@jridgewell/trace-mapping` v0.2.2 installed? It's a dependency of `@amproject/remapping`. We released v0.2.3 to fix a bug that caused an invalid source location to be returned, which the `source.originalPositionFor()` call to error as in your stack trace. If you `npm update`, you should get the fixes.

**Comment by oliversalzburg** [comment]
Hey @jridgewell! Thank you so much for the support. That definitely got me on the right track. It turned out, I had a `@babel/core` upgrade pending, which included some of your latest changes, if I'm not mistaken. After upgrading babel, the issues disappeared :)
